Styles.css

```diff
@@ -0,0 +1,199 @@
+* {
+    box-sizing: border-box; 
+    margin: 0;
+    padding: 0;   
+}
+
+html {
+    height: 100%;
+}
+
+
+.navbar {
+    display: flex;
+    justify-content: space-between;
+    padding: 20px;
+    margin: 20px;
+    background-color: antiquewhite; 
+    
+}
+
+
+h1 {
+    color: rgb(5, 147, 147);
+}
+
+h2 {
+    display: flex;
+    color: #00b39b;
+    font: 2.125rem;
+    font-weight: 400;
+    align-content: center;
+    justify-content: center;
```

Comment:
Try to group related properties together for easier readability (e.g. flex-related styles together, layout-styles together, font-styles together).